redwood-lazylayout-compose/src/commonMain/kotlin/app/cash/redwood/lazylayout/compose/LazyListState.kt

```diff
@@ -156,13 +158,38 @@ public open class LazyListState {
       }
     }
 
-    // TODO(dylan+jwilson): If we're contiguous with our previous loaded range,
-    //  don't rush to remove things from the previous range.
-    val begin = (firstIndex - preloadBeforeItemCount).coerceAtLeast(0)
-    val end = (lastIndex + preloadAfterItemCount).coerceAtMost(itemCount).coerceAtLeast(0)
+    var begin = (firstIndex - preloadBeforeItemCount).coerceAtLeast(0)
+    var end = (lastIndex + preloadAfterItemCount).coerceAtMost(itemCount).coerceAtLeast(0)
+
+    // On initial load, set lastIndex to the end of the loaded window.
+    if (lastIndex == 0) {
```

Comment:
@swankjesse Is this okay to do? I don't think it makes sense to keep (0, 0) as the range after the initial pass.

Reply:
Seems reasonable!